demo/csi/ceph-csi-plugin/example_volume.hcl

```diff
@@ -0,0 +1,22 @@
+type            = "csi"
+id              = "testvol"
+name            = "test_volume"
+# this must be strictly formatted, see README
+external_id     = "ffff-0024-01616094-9d93-4178-bf45-c7eac19e8b15-000000000000ffff-00000000-1111-2222-bbbb-cacacacacaca"
+access_mode     = "single-node-writer"
+attachment_mode = "block-device"
+plugin_id       = "ceph-csi"
+mount_options {
+  fs_type = "ext4"
+} 
+parameters {}
+secrets {
+  userID = "admin"
+  userKey = "AWBg/BtfJInSFBATOrrnCh6UGE3QB3nYakdF+g=="
```

Comment:
Just as we did with `<clusterid>` it might be a good idea to put `<userkey>` here. This is the sort of thing that will end up triggering scans for high-entropy tokens by GitHub or our security team, so redacting it will avoid false positives.